src/Akka.Persistence.PostgreSql/Journal/PostgreSqlQueryExecutor.cs

```diff
@@ -46,8 +46,9 @@ public PostgreSqlQueryExecutor(PostgreSqlQueryConfiguration configuration, Akka.
                     {Configuration.TagsColumnName} VARCHAR(100) NULL,
                     {Configuration.SerializerIdColumnName} INTEGER NULL,
                     CONSTRAINT {Configuration.JournalEventsTableName}_uq UNIQUE ({Configuration.PersistenceIdColumnName}, {Configuration.SequenceNrColumnName})
-                );";
-
+                ); CREATE INDEX IF NOT EXISTS IX_{Configuration.JournalEventsTableName}_{Configuration.SequenceNrColumnName} ON {Configuration.FullJournalTableName}  USING btree (
```

Comment:
Can we add a migration SQL script to apply this to existing journal configs?

Reply:
Sure, automatically or just a SQL file ? This index is needed for large Journals

Reply:
@jarlebh a SQL file is fine - users should apply it manually.